index.js

```diff
@@ -441,6 +444,10 @@ Router.prototype.process_params = function process_params(layer, called, req, re
  * pathname.
  *
  * @public
+ * @param {string} path (optional)
```

Comment:
Rather than making the argument's description "(optional)", can we use the JSDoc features to make it as optional? Probably a better IDE experience.